app/src/components/blocks/_delete/deleteSubmitStage/DeleteSubmitStage.tsx

```diff
@@ -148,9 +136,18 @@ function DeleteDocumentsStage({
             <form onSubmit={handleSubmit(submit)}>
                 <Fieldset id="radio-selection">
                     <Fieldset.Legend isPageHeading>
-                        Are you sure you want to permanently delete files for:
+                        You are removing the {recordType} record of:
                     </Fieldset.Legend>
-                    <div>{patientInfo}</div>
+                    <PatientSummary separator />
+
+                    <p>
+                        Once you remove this files, you can not access this record using the
+                        service. you may want to keep a copy of the paper record safe.
+                    </p>
+                    <h2>Are you sure you want to permanently remove this record?</h2>
+                    <div>
+                        <p>This can not be undone</p>
```

Comment:
```suggestion
                        <WarningText text="This can not be undone" />
```

From the Figma, looks like we should have a yellow ! mark icon here, which is packed in the WarningText component.
(might need to customise a bit if we want the text not bold)

Reply:
Implemented as suggested, awaiting response from Product on if it should be bold to maintain consistency.

Reply:
Agreed that bold is acceptable by Sarah S